_data/programs.yml

```diff
@@ -1,29 +1,29 @@
 - name: Google Summer of Docs
-  imagePath: /images/programs/gsod.svg
+  imagePath: /images/programs/SeasonOfDocs_Logo.svg
   link: /programs/gsod/2020
   new_window: false
 
 
 - name: Google Summer of Code
-  imagePath: /images/programs/gsoc.svg
+  imagePath: /images/programs/GSoC-icon.svg
   link: /programs/gsoc/2020
   new_window: false
 
 
 - name: Cloud Native Computing Foundation
-  imagePath: /images/programs/cncf-icon-white.svg
+  imagePath: /images/programs/cncf-icon-color.svg
   link:  /programs/meshery/cncf
   new_window: false
    
 
-- name: Community Bridge 2020
-  imagePath: /images/programs/CommunityBridge-circular-white.svg
+- name: LFX
+  imagePath: /images/programs/linux-foundation.svg
   link:  /programs/communitybridge/2020
```

Comment:
The link also needs to be updated to `/programs/lfx/2020`

Reply:
I tried it doing but I think there is no page as lfx/2020 so should I rename the page of community bridge as lfx and the page will also need to be updated

<img width="1438" alt="Screenshot 2021-09-03 at 6 36 17 PM" src="https://user-images.githubusercontent.com/81188792/132009883-3fcd4c4f-5a43-473e-91b2-faaf60b2f8c8.png">

so shall i rename the file and the content ?